src/main/java/be/woutschoovaerts/mollie/Client.java

```diff
@@ -26,6 +26,18 @@ public Client(String apiKey) {
         initUniRest();
     }
 
+    public Client(String apiKey, String host, int port, String username, String password) {
```

Comment:
please make the parameters obvious about proxy, so proxyHost, proxyPort, etc.

Reply:
will be done soon.